### PR TITLE
fix: Search result are replaced by original request because it is tool ong to execute - EXO-61970 - Meeds-io/meeds#758

### PR DESCRIPTION
**Pull request by GouadriaHanen**
prior to this change, the Search result of users is replaced by the original request because it takes too long to retrieve the result after this change when searching people the first fetch is abortd

### CODE REVIEW

**Review by ahamdi — COMMENTED** [review]
@GouadriaHanen Could you fix the conflicts ?

**Comment by GouadriaHanen** [comment]
> @GouadriaHanen Could you fix the conflicts ?

done!

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardList.vue [hunk 1]
```diff
@@ -165,18 +166,22 @@ export default {
...
       // Using 'limitToFetch + 1' to retrieve current user and then delete it from result
       // to finally let only 'limitToFetch' users
+      if (this.abortController) {
+        this.abortController.abort();
+      }
+      this.abortController = new AbortController();
       let searchUsersFunction;
       if (this.filter === 'connections') {
-        searchUsersFunction = this.$userService.getConnections(this.keyword, this.offset, this.limitToFetch + 1, this.fieldsToRetrieve);
+        searchUsersFunction = this.$userService.getConnections(this.keyword, this.offset, 200 + 1, this.fieldsToRetrieve, this.abortController.signal);
```
Comment (ahamdi) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardList.vue]:
```suggestion
        searchUsersFunction = this.$userService.getConnections(this.keyword, this.offset, 200 + 1, this.fieldsToRetrieve, this.abortController.signal);
```
Why did you hardcode the limitToFetch ?

Comment (GouadriaHanen) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardList.vue]: restored!

**Review by ahamdi — APPROVED** [review]

**Pull request merged**